sdk/textanalytics/cognitiveservices-textanalytics/src/recognizeEntitiesResultCollection.ts

```diff
@@ -14,14 +14,20 @@ import {
 } from "./recognizeEntitiesResult";
 import { sortByPreviousIdOrder } from "./util";
 
+/**
+ * Collection of `RecognizeEntitiesResult` objects corresponding to a batch of input documents, and
+ * annotated with information about the batch operation.
+ */
 export interface RecognizeEntitiesResultCollection extends Array<RecognizeEntitiesResult> {
   /**
-   * (Optional) if showStats=true was specified in the request this field will contain information
-   * about the request payload.
+   * Gets statistics about the input document batch and how it was processed
+   * by the service. This property will have a value when showStats is set to true
+   * in the client call.
    */
   statistics?: TextDocumentBatchStatistics;
   /**
-   * This field indicates which model was used for scoring.
+   * Gets the version of the text analytics model used by this operation on this
```

Comment:
We added a link to more information about possible values of modelVersion in the .NET docstrings.  Would it be worth adding that here as well?

Reply:
What's the link? I'm not seeing it in here: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizeLinkedEntitiesResultCollection.cs#L30

Reply:
Per our offline discussion, I added the relevant link to `TextAnalyticsOperationOptions` where the consumer specifies this property.